docs/experimental-features/host-risk-score.asciidoc

```diff
@@ -24,91 +24,248 @@ The following table shows how risk levels are applied to a host, based on the no
+. In the Host Risk Scores section, click *Enable* to install the module. 
+
+To enable host risk score from the Hosts page: 
+
+. Go to *Explore* -> *Hosts*. 
+. Select the *Host risk* tab, then click *Enable* to install the module. 
 
-. https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/host-risk-score.md#8-enable-kibana-features[Enable the `riskyHostsEnabled` feature flag].
-. In the {security-app}, go to *Dashboards* -> *Overview*, then locate the *Current host risk scores* card in the lower-right corner.
-+
 [role="screenshot"]
-image::images/host-risk-score-enable-dev-tools.png[]
-. Click *Enable via Dev Tools*. This will open the Dev Tools Console in {kib}, which will be pre-populated with the scripts required to enable the host risk score feature.
-. Click the *Play* button to run each of the scripts, in their numbered order from 1 to 11.
-+
+image::images/enable-hrs.png[Enable Host Risk Score button]
+
+To enable host risk score from a host's details page: 
+
+. Go to *Explore* -> *Hosts*. 
+. Select the *All hosts* tab, then click a host name. 
+. On the details page, scroll down to the data tables, then select the *Host risk* tab. 
+. Click *Enable* to install the module. 
+
 [role="screenshot"]
-image::images/host-risk-score-dev-tools-console.png[]
+image::images/enable-hrs-details-pg.gif[Enable host risk score from the host's details page]
+
+To enable host risk score from Console in {kib}, open a browser window and enter the following URL:
+
+[source,console]
+----------------------------------
+{KibanaURL}/s/{spaceID}/app/dev_tools#/console?load_from={KibanaURL}/s/{spaceID}/internal/risk_score/prebuilt_content/dev_tool/enable_host_risk_score
+----------------------------------
 
-[[import-host-risk-score-dashboard]]
+NOTE: If you have existing content in Console, scroll to the bottom, then paste the content. 
+
+[[upgrade-host-risk-score]]
 [discrete]
-=== Import the host risk score dashboard
+=== Upgrade host risk score 
 
-After you have set up the framework for the host risk score feature, import pre-built dashboards to analyze risky hosts in your environment:
+If you previously enabled host risk score and are upgrading to {stack} version 8.5 or later, there'll be an option to "upgrade" host risk score instead of enabling it. 
 
-. Navigate to the *Current host risk scores* card (*Dashboards -> Overview*).
-. Click *Import dashboard*.
-+
-[role="screenshot"]
-image::images/host-risk-score-import-dashboard.png[]
+Before upgrading, be mindful of the following: 
+
+* Previous host risk scores are deleted since the older data is not preserved, and new scores are created. However, if you want to retain the old host risk scores, you'll need to reindex the data _before_ upgrading the feature. Your data will then be stored in the `ml_host_risk_score_{spaceID}` and `ml_$host_risk_score_latest_{spaceID}` indices. To learn how to do this, refer to {ref}/docs-reindex.html[Reindex API]. 
```

Comment:
```suggestion
* Previous host risk scores will be deleted since the older data is not preserved, and new scores are created. However, if you want to retain the old host risk scores, you'll need to reindex the data _before_ upgrading the feature. To learn how to do this, refer to {ref}/docs-reindex.html[Reindex API]. New data will be stored in the `ml_host_risk_score_{spaceID}` and `ml_host_risk_score_latest_{spaceID}` indices. 
```